man/bundle-config.ronn

```diff
@@ -220,6 +220,8 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
 * `only_update_to_newer_versions` (`BUNDLE_ONLY_UPDATE_TO_NEWER_VERSIONS`):
    During `bundle update`, only resolve to newer versions of the gems in the
    lockfile.
+* `patch` (BUNDLE_PATCH):
+   Whether to use patch update level, by default set to false.
```

Comment:
Coming to the conversation fresh, this description doesn't give me enough information. Perhaps the option could be renamed to `prefer_patch`, and the description could read something like:

```
Prefer updating only to next patch version during updates. Makes `bundle update` calls equivalent to `bundler update --patch`.
```

Reply:
Yes, prefer_patch seems to be a better and the text you've given makes much more sense to a new user. I've updated that as well.